### PR TITLE
docs: add doc and example api model for IPv6

### PR DESCRIPTION
**Pull request by aramase**
**Reason for Change**:

- Adds example api model for IPv6 clusters
- Adds docs for IPv6 

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [x] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Review by jackfrancis — APPROVED** [review]
/lgtm

File: examples/ipv6/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# AKS Engine - IPv6 clusters
+
+## Overview
+
+AKS Engine enables you to create IPv6 Kubernetes clusters on Microsoft Azure.
+
+- IPv6 support is available for Kubernetes version 1.18.0 and later on Azure.
+- IPv6 support is in beta as of Kubernetes version 1.18 in Kubernetes community.
+
+In order to create IPv6 enabled Azure virtual networks and use standard loadbalancer with IPv6 you must first configure your subscription [as follows](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-ipv4-ipv6-dual-stack-cli#prerequisites).
```
Comment (CecileRobertMichon) [inline examples/ipv6/README.md:10]: there is no #prerequisites paragraph on that page

Comment (CecileRobertMichon) [reply examples/ipv6/README.md:10]: also nit: `as follows` implies it's described below, should be `as described [here](link)`

Comment (aramase) [reply examples/ipv6/README.md:10]: The feature is now GA, so they have removed the `prerequisites`. I'll update the doc to reflect that change.

**Pull request merged**